server/server.js

```diff
@@ -0,0 +1,44 @@
+require('dotenv').config()
+
+const express          = require('express'),
+      port             = process.env.PORT,
+      path             = require('path'),
+      webpack          = require('webpack'),
+      config           = require('../webpack.config'),
+      compiler         = webpack(config),
+      cors             = require('cors')
+      app              = express(),
+      spotifyConnector = require('./spotify-connector'),
+      request          = require('request')
+      saveAcessToken   = require('./spotify-connector'),
+      Spotify          = require('spotify-web-api-js'),
+      spotifyApi       = new Spotify(),
+      XMLHttpRequest   = require('xmlhttprequest').XMLHttpRequest
+
+app.use(cors())
+app.options('*', cors())
+app.use(require('webpack-dev-middleware')(compiler, {
```

Comment:
I też chyba czytelniej nie importować modułów gdzieś w środku funkcji.

Reply:
yup, nawet eslint mi o to krzyczal i tez mialem zrobic hehe, juz done